src/shared/icon/index.ts

```diff
@@ -2,6 +2,7 @@ import { html, LitElement } from "lit";
 import { customElement, property } from "lit/decorators.js";
 
 import styles from "./icon.css";
+//@Tristan, wanneer ik ?dev verander in ?inline dan verdwijnen de iconen. Graag advies
```

Comment:
@tristangodfrey zou je kunnen adviseren hoe we om moeten gaan met het inlinen van css in combinatie met web componenten?

Reply:
@tristangodfrey  Zie: node_modules/dmn-js/dist/assets/dmn-font/css/dmn-embedded.css

Hier hebben ze de font als data url toegevoegd. Hoe dat ook te doen met material icons: node_modules/material-symbols/outlined.css

Reply:
@tristangodfrey Het is een chrome "bug". Ik heb een work around gemaakt door de font-face declaratie toe te voegen aan de dom.